initializr-service/src/main/resources/application.yml

```diff
@@ -951,6 +958,67 @@ initializr:
           description: Hystrix circuit breaker on Pivotal Cloud Foundry
           groupId: io.pivotal.spring.cloud
           artifactId: spring-cloud-services-starter-circuit-breaker
+    - name: Azure
+      bom: azure
+      versionRange: 1.5.4.RELEASE
+      content:
+        - name: Azure Active Directory
+          id: active-directory
+          groupId: com.microsoft.azure
+          artifactId: azure-ad-integration-spring-boot-starter
+          description: Spring Security integration with Azure Active Directory for authentication
+          links:
+            - rel: guide
+              href: https://github.com/Microsoft/azure-spring-boot/tree/master/azure-samples/azure-ad-integration-spring-boot-starter-sample
+              description: Sample code for using the starter
+            - rel: reference
+              href: https://github.com/Microsoft/azure-spring-boot/tree/master/azure-starters/azure-ad-integration-spring-boot-starter
+              description: Source code and README for the starter
+        - name: Azure Cosmos DB (DocumentDB)
```

Comment:
We don't believe this starter should have a dedicate entry at this point. We can reconsider based on usage metrics.